src/components/ZestyExplorer/ZestyExplorer.tsx

```diff
@@ -101,16 +101,15 @@ export const ZestyExplorer = ({ content = {} }: any) => {
    }
 
    return (
-      <div id={"zestyExplorer"} style={zestyWrapper}>
+      <Box id={"zestyExplorer"} sx={zestyWrapper}>
          <Helmet>
             <script src="https://cdn.jsdelivr.net/gh/zesty-io/fetch-wrapper@latest/dist/index.min.js" />
          </Helmet>
          <ThemeProvider theme={getTheme("light", themeToggler)}>
             <CssBaseline />
             {/* ZESTY LOGO  bottom right*/}
             {!open && (
-               <button
-                  type="button"
+               <div
```

Comment:
Use `<Box>`

Reply:
or `<Button>` ?

Reply:
Button and we need to override the hover effect because it become transparent